docs/jwst/assign_wcs/arguments.rst

```diff
@@ -0,0 +1,30 @@
+Step Arguments
+==============
+
+The ``assign_wcs`` step has the following optional arguments to control
+the behavior of the processing.
+
+``--sip_approx`` (boolean, default=True)
+  A flag to enable the computation of a SIP approximation for
+  imaging modes.
+
+``--sip_max_pix_error`` (float, default=0.1)
+  Maximum error for the SIP forward fit, in units of pixels.
```

Comment:
You can add that `sip_max_pix_error` is ignored if `sip_degree` is a number, i.e. not None.

Reply:
Done.